src/condition.cpp

```diff
@@ -1785,6 +1789,15 @@ std::function<double( dialogue & )> conditional_t::get_get_dbl( J const &jo )
                 bodypart_id bid = bp.value_or( get_bp_from_str( d.reason ) );
                 return d.actor( is_npc )->get_cur_hp( bid );
             };
+        } else if( checked_value == "hp_max" ) {
```

Comment:
This is the legacy way to add accesors, you should instead place it in a math function in: math_parser_diag.h there are similar examples in there for stuff, armor has examples of getting the BP.

Reply:
it's moved  to math_parser_diag